{
    "TDGPlannedOutageMessageHandlerLambdaFunction": {
        "type": "AWS::Serverless::Function",
        "account_name": "aws-1",
        "invokes": [],
        "invoked_by": [],
        "invoked_by_external": [
            "S3"
        ]
    },
    "LambdaInvokePermission": {
        "type": "AWS::Lambda::Permission",
        "account_name": "aws-1",
        "invokes": [],
        "invoked_by": [],
        "invoked_by_external": []
    },
    "TDGPlannedOutageSendReportLambda": {
        "type": "AWS::Serverless::Function",
        "account_name": "aws-1",
        "invokes": [],
        "invoked_by": [
            {
                "name": "TDGPlannedOutageSendReportSQS",
                "type": "AWS::SQS::Queue",
                "account_name": "aws-1"
            }
        ],
        "invoked_by_external": []
    },
    "TDGPlannedOutageSendReportSQS": {
        "type": "AWS::SQS::Queue",
        "account_name": "aws-1",
        "invokes": [
            {
                "name": "TDGPlannedOutageSendReportLambda",
                "type": "AWS::Serverless::Function",
                "account_name": "aws-1"
            }
        ],
        "invoked_by": [],
        "invoked_by_external": []
    },
    "TDGPlannedOutageSendReportSQSEventPolicy": {
        "type": "AWS::SQS::QueuePolicy",
        "account_name": "aws-1",
        "invokes": [],
        "invoked_by": [],
        "invoked_by_external": []
    }
}